pulp_ansible/app/viewsets.py

```diff
@@ -60,9 +63,33 @@ class RoleViewSet(SingleArtifactContentUploadViewSet):
     filterset_class = RoleFilter
 
 
+class CollectionFilter(BaseFilterSet):
+    """
+    FilterSet for Ansible CollectionVersions.
+    """
+
+    namespace = filters.CharFilter(field_name="namespace")
+    name = filters.CharFilter(field_name="name")
+
+    class Meta:
+        model = Collection
+        fields = ["namespace", "name"]
+
+
+class CollectionViewset(NamedModelViewSet, mixins.RetrieveModelMixin, mixins.ListModelMixin):
+    """
+    Viewset for Ansible Collections.
+    """
+
+    endpoint_name = "ansible/collections"
```

Comment:
Does this still nest under `/pulp/api/v3/` ?

Reply:
Yes

Reply:
Thanks!